src/docs/releases/2.0.0.md

```diff
@@ -1,6 +1,730 @@
 # Orchard Core 2.0.0
 
-Release date: Not yet released
+Release date: September 9, 2024. 
+
+🎉 **Orchard Core 2.0 Is Here – Elevate Your Projects to the Next Level!** 🎉
+
+We are thrilled to announce the **second major release** of Orchard Core! This release is packed with **performance improvements**, **bug fixes**, and a variety of **enhancements** that make it our best version yet. Whether you're building a new project or maintaining an existing one, Orchard Core 2.0 brings stability and power to your fingertips.
+
+### 🚀 Why Upgrade to Orchard Core 2.0?
+
+- **Enhanced Performance**: Your sites and applications will run faster, smoother, and more efficiently.
+- **Rock-Solid Stability**: We've squashed numerous bugs, ensuring a more reliable and enjoyable development experience.
+- **Seamless Development**: By sticking to version control best practices, we've ensured **no breaking changes** in any patch or minor release, allowing developers to confidently create modules and libraries that will work across the entire 2.x series.
+
+
+### 🎯 Module and Library Developers, Take Note!
+
+Want to create a module or library for Orchard Core 2.0? We've made it easier than ever to build with confidence. Simply set your dependencies to `[2.0, 3.0)` to ensure full compatibility with future updates, without worrying about breaking changes.
+
+## What's Changed
```

Comment:
This could be auto-generated in GitHub release

Reply:
It was :)